.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
annoyingly flake8 doesn't support reading the config from pyproject.toml so we have to move back to a separate file